packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -351,6 +351,22 @@ const ruleset: any = {
       },
     },
 
+    // RPC Code: RPC-Get-V1-14
+    GetMustNotBeLRO: {
```

Comment:
>GetMustNotBeLRO [](http://example.com/codeflow?start=4&length=15)

GetOperationMustNotBeLongRunning ?

Reply:
I found this big while naming the files with hyphen "get-operations-must-not-be-long-running.*", but still considered and changed.